crypto/bn256/google/constants.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2012 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bn256
+
+import (
+	"math/big"
+)
+
+func bigFromBase10(s string) *big.Int {
+	n, _ := new(big.Int).SetString(s, 10)
+	return n
+}
+
+// u is the BN parameter that determines the prime: 1868033³.
+var u = bigFromBase10("4965661367192848881")
```

Comment:
May I know why those value of constant variables are not same as the cloudflare/google branch, for example, **var u = bigFromBase10("6518589491078791937")**. Based on the comment of this line, 1868033³ also equal to 6518589491078791937, not 4965661367192848881